src/on-event.js

```diff
@@ -0,0 +1,44 @@
+function displayMenu() {
+  function insertElement(elem, className) {
+    const element = document.createElement(elem);
+    element.className = className;
+    return element;
+  }
+
+  const hamburgerBtn = document.querySelector(".hamburger");
+  const navLinksWrapper = document.querySelector(".hide-mobile");
+  const cross = insertElement("img", "cross");
+  const li = insertElement("li", "cross-icon-wrapper");
+
+  function onClickDisappear() {
+    navLinksWrapper.style.display = "none";
+    li.remove();
+    cross.remove();
```

Comment:
- [ ] Kindly follow the font size and text color for mobile menu options as per the Figma design.

Figma design:
![2022-05-10 (3)](https://user-images.githubusercontent.com/70269507/167639467-656b2e17-6322-40e9-9546-ebb652028774.png)
Your design: 
![2022-05-10 (2)](https://user-images.githubusercontent.com/70269507/167639474-0d5b2532-37f0-4365-9e0e-665166e411df.png)